src/status.js

```diff
@@ -0,0 +1,13 @@
+import updateStorage from './storage';
+
+function checkStatus(sortList, e) {
```

Comment:
* I get errors in the console when i try this checkStatus functionality please 🙏 kindly make the necessary adjustments 
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
![image](https://user-images.githubusercontent.com/52497246/126022149-91733a25-b8ee-4113-b051-1cb68f521be1.png)